### ISSUE TITLE
Floppy image boot fails on virtual box 7.0.2 with "No support for int13h extensions"

### ISSUE THREAD

**Issue by hugo-hur** [rust-osdev/bootloader#273]
Hi!

I cannot get my test code to boot on virtual box. Just "No support for int13h extensions" message is displayed.

I assume this does not even go to the point where my code would matter?

**Comment by phil-opp** [comment]
Yeah, this happens in the bootloader before your kernel is even loaded. I don't have much experience with VirtualBox, but the [int 13h](https://en.wikipedia.org/wiki/INT_13H) extensions are normally well-supported on modern (virtual) machines.

Which version of VirtualBox are you using?

**Comment by arnavthorat78** [comment]
This occurs in VMware too for me. Is there a fix for this yet?
